xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -597,6 +606,8 @@ The property name should already be escaped when the instance of <xref:System.Te
 
 ## Remarks
 
+The maximum allowed size of parameter `bytes` is currently 125 MB (131072000 bytes). Exceeding this limit will result in an <xref:System.ArgumentException> being thrown.
```

Comment:
I don't think is accurate and the limit is 125_000_000 bytes (not 125 MB/131072000 bytes). 

```suggestion
The maximum allowed size of the binary data to write as Base64 is 125_000_000 bytes (or ~125MB). Exceeding this limit will result in an <xref:System.ArgumentException> being thrown.
```

Same feedback for the rest of the remarks.